### PR TITLE
OCPQE-18090: combine nested virtualization and OS disk type & size testing

### PR DESCRIPTION
**Pull request by jianli-wei**
- update chain `cucushift-installer-rehearse-gcp-ipi-customer-vpc-provision`, to insert OS disk type & size settings into install-config
- rename and update workflow `cucushift-installer-rehearse-gcp-ipi-nested-virtualization-osdisk-type-size` to insert post-installation checks of OS disk type & size
- remove verification test `gcp-ipi-osdisk-type-size-f28`, which is combined into `gcp-ipi-nested-virtualization-osdisk-type-size-f28`

### CODE REVIEW

**Comment by jianli-wei** [comment]
periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-nested-virtualization-osdisk-type-size-f28

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/nested-virtualization/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-workflow.yaml [hunk 1]
```diff
@@ -4,11 +4,15 @@ workflow:
     pre:
     - chain: cucushift-installer-rehearse-gcp-ipi-customer-vpc-provision
     - ref: cucushift-installer-check-gcp-nested-virtualization
+    - ref: cucushift-installer-check-gcp-disktype-compute
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/nested-virtualization/cucushift-installer-rehearse-gcp-ipi-nested-virtualization-workflow.yaml:7]: Shall we rename the install workflow name accordingly to indicate `disk-type-size`

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml [hunk 2]
```diff
@@ -115,8 +115,8 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-users-static
-- as: gcp-ipi-nested-virtualization-f28
-  cron: 55 20 8 * *
+- as: gcp-ipi-nested-virtualization-osdisk-type-size-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml:118]: shall we update the frequency to `f14` for more coverage?

Comment (jianli-wei) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml:118]: I think `f28` is enough, according to our test history. Thanks! 

Test Case | Last 10 Runs Results | Comment
-- | -- | --
[OCP-30790 Install cluster with valid disktype and disksize specified](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-30790) | All Passed| 
[OCP-32001 Install a cluster with nested virtualization enabled](https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-32001) | 9 Passed 1 Failed | The failed one was during "Full Functional Testing_OCP_4.9_Installer_Before_GA"

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-nested-virtualization-osdisk-type-size-f28

**Pull request merged**